src/code/z_en_item00.c

```diff
@@ -776,10 +776,10 @@ void EnItem00_Update(Actor* thisx, GlobalContext* globalCtx) {
 
     if ((this->actor.params == ITEM00_SHIELD_DEKU) || (this->actor.params == ITEM00_SHIELD_HYLIAN) ||
         (this->actor.params == ITEM00_TUNIC_ZORA) || (this->actor.params == ITEM00_TUNIC_GORON)) {
-        f32 newUnkBC = Math_CosS(this->actor.shape.rot.x) * 37.0f;
+        f32 cos = Math_CosS(this->actor.shape.rot.x) * 37.0f;
 
-        this->actor.shape.yOffset = newUnkBC;
-        if (newUnkBC >= 0.0f) {
+        this->actor.shape.yOffset = cos;
+        if (cos >= 0.0f) {
             this->actor.shape.yOffset = this->actor.shape.yOffset;
         } else {
             this->actor.shape.yOffset = -this->actor.shape.yOffset;
```

Comment:
I would name `cos` `yOffset` and the condition after looks a lot like `this->actor.shape.yOffset = ABS(this->actor.shape.yOffset)`